examples/test_examples.py

```diff
@@ -435,18 +435,20 @@ def test_text_to_image_checkpointing(self):
             pipe(prompt, num_inference_steps=2)
 
             # check checkpoint directories exist
-            self.assertTrue(os.path.isdir(os.path.join(tmpdir, "checkpoint-2")))
-            self.assertTrue(os.path.isdir(os.path.join(tmpdir, "checkpoint-4")))
+            self.assertEqual(
+                set(os.listdir(os.path.join(tmpdir, "checkpoints"))),
+                {"checkpoint_0", "checkpoint_1"},
+            )
```

Comment:
Changed names of checkpoint names.

The set equality is also a better check because it checks these are the _only_ checkpoints, not just that they exist